bentoml/models.py

```diff
@@ -44,19 +43,155 @@ def delete(
 
 @inject
 def import_model(
-    path: str, *, _model_store: "ModelStore" = Provide[BentoMLContainer.model_store]
+    path: str,
+    input_format: t.Optional[str] = None,
+    *,
+    protocol: t.Optional[str] = None,
+    user: t.Optional[str] = None,
+    passwd: t.Optional[str] = None,
+    params: t.Optional[t.Dict[str, str]] = None,
+    subpath: t.Optional[str] = None,
+    _model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ) -> Model:
-    return Model.from_fs(fs.open_fs(path)).save(_model_store)
+    """
+    Import a bento model exported with :code:`bentoml.models.export_model`. To import a model saved
+    with a framework, see the :code:`save` function under the relevant framework, e.g.
+    :code:`bentoml.sklearn.save`.
+
+    Examples:
+
+    .. code-block:: python
```

Comment:
```suggestion
    .. code-block:: python
    
```
This fixed the formatting, currently this code block is not showing up: https://bentoml--2237.org.readthedocs.build/en/2237/api/stores.html